File: Data Structures (Java & C++)/Project 7/Project 7 C++/Node.cpp
//
//  Node.cpp
//  Project 7 C++
//
//  Created by Alexandru Gutu on 11/23/14.
//  Copyright (c) 2014 Alexandru Gutu. All rights reserved.
//

#include "Node.h"

Node::Node(int data)
{
	this->data = data;
}

Node::Node(int data, Node* next)
{
	this->data = data;
	this->nextNode = next;
}

int Node::getData()
{
	return data;
}

Node* Node::getNext()
{
	return nextNode;
}

void Node::setNext(Node* next)
{
	this->nextNode = next;
}

File: Object Oriented Java (Java)/Assignment 4/Node.java
/**
 * Created by alexandrugutu on 3/6/14.
 */
public class Node
	{
		private int data;
		private Node next;

		public Node(int data)
		{
			this.data = data;
		}

		public Node(int data, Node n)
		{
			this.data = data;
			next = n;
		}

		public int getData()
		{
			return data;
		}

		public Node getNext()
		{
			return next;
		}

		public void setNext(Node n)
		{
			next = n;
		}




	}